templates/funcs.go

```diff
@@ -205,7 +211,12 @@ if {{ .SliceName }}Length > 100 {
 
 {{ .SliceName }} := make(utils.{{ .SliceType }}, {{ .SliceName }}Length)
 for i := 0; i < {{ .SliceName }}Length; i++ {
-	{{ .SliceName }}[i] = {{ .Getter }}[i]
+	{{ if .IsStars }}
+		{{ .SliceName }}[i] = "***"	
+		continue
```

Comment:
I think you're mistaking the template "if" with Go "ifs", so the `continue` here is actually useless